lib/systems/architectures.nix

```diff
@@ -387,12 +387,12 @@ rec {
 
     # x86_64 AMD
     # TODO: fill this (need testing)
```

Comment:
Can this line be removed then?

Reply:
I only added the generic microarchitecture level inferiors for these AMD CPUs.

The relationships between more specific CPU architectures are still missing.

Reply:
I modified the comment to reflect this.